src/app/components/Button/index.tsx

```diff
@@ -67,12 +67,15 @@ const Button = forwardRef(
         onClick={onClick}
         disabled={disabled}
       >
-        {loading && (
-          <div className={direction === "row" ? "mr-2" : ""}>
-            <Loading color={primary ? "white" : "black"} />
+        {(loading || icon) && (
+          <div
+            className={classNames(
+              direction === "row" ? "mr-2 flex justify-center min-w-[24px]" : ""
+            )}
```

Comment:
@reneaaron this change in styles is necessary, because having an icon and loading state results in a very small horizontal jump of the text, due to the rotating loading icon and the icon itself not having the same width. I'm kind of unsatisfied with this change, but I found no better way in preventing this rather annoying UX problem. Please have a look and let me know. You can revert by "show diff" branch master and revert the changes to see what I mean.